diff_drive_controller/include/diff_drive_controller/diff_drive_controller.hpp

```diff
@@ -25,6 +25,7 @@
 #include <queue>
 #include <string>
 #include <vector>
+#include <stdio.h>
```

Comment:
thanks for the fix *however* in C++ it is not recommended to use C headers
you could use `#include <cstdio>` for replacing this

but I think a more appropriate include that also fixes this is
```suggestion
#include <cstddef>
```

Reply:
also... the error message is clear about this missing from *a different header*
```
/home/noeel/dev_ws/webots_foxy/src/ros2_controllers/diff_drive_controller/include/diff_drive_controller/rolling_mean_accumulator.hpp:37:41: fout: expected ‘)’ before ‘rolling_window_size’
```
you should only add missing dependencies to the files that miss them ;)